uaber-api/uaber/routers/trip.py

```diff
@@ -0,0 +1,19 @@
+from fastapi import APIRouter
+
+from uaber.models import Driver, Trip, Refugee, BorderCrossingPoint, DestinationPoint
+from uaber.services import trip as trip_service
+
+router = APIRouter(
+    prefix='/trip',
```

Comment:
Because this API is for managing of collection resource, you should change this prefix to `/trips`. More details you can find under [this](https://restfulapi.net/resource-naming/) url.